pom.xml

```diff
@@ -82,16 +82,14 @@
     <properties>
         <copyright>2010 - 2022</copyright>
         <maven.build.timestamp.format>yyyy-MM-dd HH:mm:ss</maven.build.timestamp.format>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
-        <maven.compiler.testSource>1.8</maven.compiler.testSource>
-        <maven.compiler.testTarget>1.8</maven.compiler.testTarget>
+        <maven.compiler.source>8</maven.compiler.source>
+        <maven.compiler.target>8</maven.compiler.target>
+        <maven.compiler.testSource>8</maven.compiler.testSource>
+        <maven.compiler.testTarget>8</maven.compiler.testTarget>
         <maven.min-version>3.3.9</maven.min-version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.build.resourceEncoding>UTF-8</project.build.resourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <signature.artifact>java18</signature.artifact>
-        <signature.version>1.0</signature.version>
```

Comment:
What is the benefit of moving properties to a profile? They are harmless in the main POM and easier to maintain if they are all together in one spot. We have the JNA and SLF4J versions in the parent pom too, even though they are only used by child modules.

Reply:
Also, will dependabot maintain versions in profiles? I don’t think so… by removing the main execution from the default POM it hides these.  Not that they will ever change, of course.  They may as well be hard coded without properties.